lib/IRGen/GenExistential.cpp

```diff
@@ -1355,6 +1357,36 @@ createErrorExistentialTypeInfo(IRGenModule &IGM,
                                       refcounting);
 }
 
+llvm::Type *TypeConverter::getExistentialType(unsigned numWitnessTables) {
+  auto &type = OpaqueExistentialTypes[numWitnessTables];
```

Comment:
Could you please use the explicit type here.  The reference makes this confusing - the reference indicates that the value is non-null, and the type check on the next line doesn't make sense until you realise that the value that you have a reference to is a pointer.

Reply:
Done.